docs/group_quick_starts/gc_account_deletion.rst

```diff
@@ -0,0 +1,21 @@
+Account Deletion
+================================
+
+You can now delete your account without asking help from Solace Cloud support. However, this feature is limited to standard accounts only.
+
+In the below image, under Account tab, the Advance Setting link is been added to provide option for deleting account.
+
+.. figure:: ../img/delete_account_advanced_settings.png
+
+
+.. figure:: ../img/delete_account_popup.png
+
+After clicking the Delete Account button you would get a confirmation pop up which will inform you about all the impacts post deletion of your account.
+
+.. figure:: ../img/delete_account_confirmation_popup.png
+
+You would require to enter your password to ensure that it is an intentional deletion and voila, the account is deleted.
+
+.. figure:: ../img/delete_account_success_msg.png
+
+Once the account is deleted, all of your messaging services will be terminated, your outstanding bill will be emailed to you and your credit card would be charged as per the usage.
```

Comment:
> Once the account is deleted, all of your messaging services will be terminated, your outstanding bill will be emailed to you, your credit card will be charged, and you will be logged out of the system.

>If you require any assistance after your account has been deleted, please contact our Support team.

@mikesmith40 , comments?

Reply:
@ianmwong how about this one:

Once the account is deleted, all of your messaging services will be terminated and you will be logged out of the system. Your outstanding bill will be emailed to you and your credit card would be charged. 

If you require further assistance after your account has been deleted, please contact our Support team.